contracts/ForkDelta.sol

```diff
@@ -105,18 +151,54 @@ contract ForkDelta {
     emit Withdraw(token, msg.sender, amount, tokens[token][msg.sender]);
   }
 
+  /**
+  * Retrieves the balance of a token based on a user address and token address.
+  * @param token Ethereum contract address of the token 
+  * @param user Ethereum address of the user
+  * @return the amount of tokens on the exchange for a given user address
+  */
   function balanceOf(address token, address user) public constant returns (uint) {
     return tokens[token][user];
   }
 
+  /**
+  * Note: tokenGet & tokenGive can be the Ethereum contract address.
+  * Stores the active order inside of the contract.
+  * Emits an Order event.
+  * @param tokenGet Ethereum contract address of the token to receive
+  * @param amountGet uint amount of tokens being received
+  * @param tokenGive Ethereum contract address of the token to give
+  * @param amountGive uint amount of tokens being given
+  * @param expires uint of block number when this order should expire
+  * @param nonce arbitrary random number
+  */
   function order(address tokenGet, uint amountGet, address tokenGive, uint amountGive, uint expires, uint nonce) public {
     bytes32 hash = sha256(this, tokenGet, amountGet, tokenGive, amountGive, expires, nonce);
     orders[msg.sender][hash] = true;
     emit Order(tokenGet, amountGet, tokenGive, amountGive, expires, nonce, msg.sender);
   }
 
+  /**
+  * Note: tokenGet & tokenGive can be the Ethereum contract address.
```

Comment:
Can we generally have the notes below the definition of what the function does?

Reply:
Yeah, makes more sense that way.